ttnn/cpp/ttnn/tensor/types.hpp

```diff
@@ -65,6 +65,8 @@ class SimpleShape {
     static constexpr auto attribute_names = std::forward_as_tuple("value");
     auto attribute_values() const { return std::forward_as_tuple(this->value); }
 
+    SimpleShape get_physical_shape(Layout layout, const std::optional<Tile>& tile = std::nullopt) const;
```

Comment:
Please make a free function. I don't want anybody to think we promote this as a part of the SimpleShape interface